meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx

```diff
@@ -105,15 +135,23 @@ export const ConfigManifestOAuthFlowComponent = withTranslation()(
 						// If this is set, we have completed the authentication procedure.
 						// A reset button is provided to begin the flow again if authorization is revoked by the user.
 						<div className="mod mvs mhs">
-							<button className="btn btn-secondary" onClick={() => this.resetAuthentication()}>
-								{t('Reset Authentication')}
+							<button
+								className="btn btn-secondary btn-tight"
+								onClick={() => this.resetAppCredentials()}
+								style={{ marginRight: '10px' }}
```

Comment:
Could this be changed to a utility class instead? I think `mrs` utility class has the same effect.